.github/workflows/build.yml

```diff
@@ -0,0 +1,53 @@
+name: build
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches:
+
+concurrency:
+  group: ${{ github.head_ref }}
+  cancel-in-progress: true
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        platform:
+          - iOS
+          - tvOS
+          #- watchOS
```

Comment:
I can't enable this until they rev macOS. It's pinned at 10.15, and thus not the latest Xcode.
macOS 11 is in a limited beta